spec/dummy/config/initializers/secret_token.rb

```diff
@@ -1,7 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-Dummy::Application.config.secret_token = '16a85a222f50e7c811e690a75a82762c72861d5c5afb50e89b490c4186f2f4e3cc95f6c94c8529b6cfd32e3705b71af18dcedb5445cdd9e1fd64c293c1fd92d6'
```

Comment:
Similar to `metasploit-model` and `metasploit-concern`, recommend to use an `ENV` and `.env` file, and update the documentation.

Reply:
This is a dummy/volatile app that's just used for testing; I believe as Dean calls out in the description we should be configuring `secret_key_base` these days - and this value is now generally configured via rail's `credentials`:

```
$ rails new test_app
$ rails credentials:show 
```

```yml
# aws:
#   access_key_id: 123
#   secret_access_key: 345

# Used as the base secret for all MessageVerifiers in Rails, including the one protecting cookies.
secret_key_base: 31150f1534fd63d2bfe913099949aeb8b56db4a09712ea18b814a714a38f6a4b7cfeaa3025a3662ee77ee46b5bede64dab83ec48e90663c92387073afa361dc6
```

In this scenario though, we can just get rid of it entirely :+1: